screens/CreateEventScreen.jsx

```diff
@@ -0,0 +1,414 @@
+              <View style={styles.formButtonContainer}>
+                <Text style={styles.formHeaderText}>Wheelchair Accessible?</Text>
+                <Switch
+                  style={styles.formResponseContainer}
+                  value={wheelchair}
+                  onValueChange={toggleWheelchair}
+                />
+              </View>
+              <View style={styles.formButtonContainer}>
+                <Text style={styles.formHeaderText}>Hearing Assistance?</Text>
+                <Switch
+                  style={styles.formResponseContainer}
+                  value={hearingAssistance}
+                  onValueChange={toggleHearingAssistance}
+                />
+              </View>
+            </View>
+          </View>
+
+          <View style={styles.formSectionContainer}>
+            <Text style={styles.formSectionText}>Logistical Information</Text>
+            <View style={styles.formBlockContainer}>
+              <Text style={styles.formHeaderText}>Minimum Ticket Price</Text>
+              <View style={styles.formResponseContainer}>
+                <TextInput
+                style={styles.formResponseText}
+                placeholder="e.g. $25.00"
+                value={ticketPrice}
+                textAlign={'left'}
+                onChangeText={(text) => setTicketPrice(text)}
+                />
+              </View>
+            </View>
+            <View style={styles.formBlockContainer}>
+              <Text style={styles.formHeaderText}>Ticket Link</Text>
+              <View style={styles.formResponseContainer}>
+                <TextInput
+                style={styles.formResponseText}
+                placeholder="e.g. https://example.com"
+                value={ticketLink}
+                textAlign={'left'}
+                onChangeText={(text) => setTicketLink(text)}
+                />
+              </View>
+              <View style={styles.buttonRow}>
+                <View style={styles.formButtonContainer}>
+                  <Text style={styles.formHeaderText}>Intermission?</Text>
+                  <Switch
+                    style={styles.formResponseContainer}
+                    value={intermission}
+                    onValueChange={toggleIntermission}
+                  />
+                </View>
+                <View style={styles.formButtonContainer}>
+                  <Text style={styles.formHeaderText}>Ticket Required?</Text>
+                  <Switch
+                    style={styles.formResponseContainer}
+                    value={ticketRequired}
+                    onValueChange={toggleTicketRequired}
+                  />
+                </View>
+              </View>
+            </View>
+          </View>
+
+          <View style={styles.formSectionContainer}>
+            <Text style={styles.formSectionText}>Cast & Creatives</Text>
+            <View style={styles.container}>
+            <View style={styles.formResponseContainer}>
```

Comment:
Maybe a flatlist here? We should also probably be letting them delete cast blocks. Re-ordering can be saved for long after MVP. No idea how to implement that.